Dockerfile.autobuild

```diff
@@ -21,30 +42,24 @@ RUN groupadd -g 2300 tmpgroup \
  && usermod -u 1000 www-data \
  && groupdel tmpgroup
 
-RUN mkdir /data
-RUN chown -R www-data:www-data /data
-
+# Install nginx
 RUN apt-get update \
-    && apt-get -y install nginx nginx-extras pwgen \
+    && apt-get -y  install nginx \
```

Comment:
`pwgen` is required if you haven't created `local_settings.py` because the django-secret is generated by this command-line tool during bootstrap stage for project.

Reply:
Thanks, I will add it.